src/main/java/forum/entity/Thread.java

```diff
@@ -0,0 +1,84 @@
+package forum.entity;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="thread")
+public class Thread {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id")
+	private int id;
+
+	@Column(name = "title")
+	private String title;
+	
+	@Column(name="content")
+	private String content;
+	
+	
+	@ManyToOne(cascade = { CascadeType.PERSIST, CascadeType.MERGE,
+							CascadeType.DETACH, CascadeType.REFRESH })
+	@JoinColumn(name = "category_id")
+	private Category category;
+	
+	public Thread() {
+		
+	}
+
+	public Thread(int id, String title) {
+		this.id = id;
+		this.title = title;
```

Comment:
Hmm so you mean content it optional? Hmm does it make sense that user can create thread with only title?

Reply:
I actually forgot to add the content variable to the constructor. My bad! (I first created the Thread entity without the content variable by accident and added it later)